ext/standard/basic_functions.stub.php

```diff
@@ -153,11 +153,11 @@ function array_merge(array ...$arrays): array {}
 
 function array_merge_recursive(array ...$arrays): array {}
 
-function array_replace(array $array1, array ...$arrays): array {}
+function array_replace(array $array, array ...$with): array {}
```

Comment:
Would `$substitutions` work in this context?

Reply:
Neither of these work right now, but supposing variadics and named params ever play nice:

```php
$res = array_replace(array=$arr, with=$maps);
```
vs
```php
$res = array_replace(array=$arr, substitutions=$maps);
```

I can see either working.  No strong preference.  Anyone else want to weigh in?